DESCRIPTION

```diff
@@ -52,5 +52,5 @@ Encoding: UTF-8
 Remotes: github::statnet/statnet.common@master,
     github::statnet/network@master,
     github::statnet/networkDynamic@master,
-    github::statnet/ergm@master,
-    github::statnet/ergm.multi@master
+    github::statnet/ergm@basis,
+    github::statnet/ergm.multi@basis
```

Comment:
@chad-klumb , will the `@basis` bits be reverted after the PR for `ergm` is merged?

Reply:
they should be; I merged the ergm PR and pushed an update to the `tergm` PR replacing `@basis` with `@master`